perception/multi_object_tracker/include/multi_object_tracker/utils/utils.hpp

```diff
@@ -78,6 +80,129 @@ inline bool isLargeVehicleLabel(const uint8_t label)
   return label == Label::BUS || label == Label::TRUCK || label == Label::TRAILER;
 }
 
+/**
+ * @brief Get the Nearest Corner or Surface from detected object
+ *
+ * @param object
+ * @return int index
+ */
+inline int getNearestCornerSurface(
```

Comment:
memo
- easy to understand `getNearestCornerOrSurface`

Reply:
Ok. This is also fixed in latest version.